datafusion/datasource/src/file_scan_config.rs

```diff
@@ -528,7 +528,12 @@ impl DataSource for FileScanConfig {
         match t {
             DisplayFormatType::Default | DisplayFormatType::Verbose => {
                 let schema = self.projected_schema();
-                let orderings = get_projected_output_ordering(self, &schema);
+                // Remove None from projected output ordering
```

Comment:
I think we should display the `None` cases too, otherwise this is going to be pretty confusing. Imagine a projection that leads to `Some, None, Some` and you now gonna display 2 orderings (for 3 partitions). From that you cannot tell if it was `None, Some, Some`, `Some, None, Some`, or `Some, Some, None`.